[2025-06-12T19:29:10.987+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: procesamiento_datos_autopia_dag.trigger_entrenamiento manual__2025-06-12T19:29:06.954114+00:00 [queued]>
[2025-06-12T19:29:11.019+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: procesamiento_datos_autopia_dag.trigger_entrenamiento manual__2025-06-12T19:29:06.954114+00:00 [queued]>
[2025-06-12T19:29:11.020+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-12T19:29:11.083+0000] {taskinstance.py:2191} INFO - Executing <Task(TriggerDagRunOperator): trigger_entrenamiento> on 2025-06-12 19:29:06.954114+00:00
[2025-06-12T19:29:11.099+0000] {standard_task_runner.py:60} INFO - Started process 519 to run task
[2025-06-12T19:29:11.115+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'procesamiento_datos_autopia_dag', 'trigger_entrenamiento', 'manual__2025-06-12T19:29:06.954114+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/procesamiento_datos_autopia_dag.py', '--cfg-path', '/tmp/tmp1kytb75e']
[2025-06-12T19:29:11.125+0000] {standard_task_runner.py:88} INFO - Job 96: Subtask trigger_entrenamiento
[2025-06-12T19:29:11.385+0000] {task_command.py:423} INFO - Running <TaskInstance: procesamiento_datos_autopia_dag.trigger_entrenamiento manual__2025-06-12T19:29:06.954114+00:00 [running]> on host 24edc8b790cf
[2025-06-12T19:29:11.784+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fede' AIRFLOW_CTX_DAG_ID='procesamiento_datos_autopia_dag' AIRFLOW_CTX_TASK_ID='trigger_entrenamiento' AIRFLOW_CTX_EXECUTION_DATE='2025-06-12T19:29:06.954114+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-12T19:29:06.954114+00:00'
[2025-06-12T19:29:14.972+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/google/cloud/links/dataproc.py:79 AirflowProviderDeprecationWarning: This DataprocLink is deprecated.
[2025-06-12T19:29:14.976+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/google/cloud/links/dataproc.py:128 AirflowProviderDeprecationWarning: This DataprocListLink is deprecated.
[2025-06-12T19:29:16.913+0000] {trigger_dagrun.py:210} INFO - Waiting for entrenamiento_modelos_regresion on 2025-06-12 19:29:11.789107+00:00 to become allowed state ['success'] ...
[2025-06-12T19:30:13.773+0000] {trigger_dagrun.py:223} INFO - entrenamiento_modelos_regresion finished with allowed state success
[2025-06-12T19:30:13.814+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=procesamiento_datos_autopia_dag, task_id=trigger_entrenamiento, execution_date=20250612T192906, start_date=20250612T192910, end_date=20250612T193013
[2025-06-12T19:30:13.939+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-12T19:30:14.010+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
